qutebrowser/config/configtypes.py

```diff
@@ -1223,14 +1224,55 @@ class AcceptCookies(BaseType):
                                ('never', "Don't accept cookies at all."))
 
 
-class ConfirmQuit(BaseType):
+class ConfirmQuit(List):
 
     """Whether to display a confirmation when the window is closed."""
 
+    typestr = 'string-list'
+
     valid_values = ValidValues(('always', "Always show a confirmation."),
                                ('multiple-tabs', "Show a confirmation if "
                                                  "multiple tabs are opened."),
+                               ('downloads', "show a confirmation if downloads "
+                                             "are running"),
                                ('never', "Never show a confirmation."))
+    # Values that can be combined with commas
+    combinable_values = ('multiple-tabs', 'downloads')
+
+    def validate(self, value):
+        values = self.transform(value)
+        # Never can't be set with other options
+        if 'never' in values and len(values) > 1:
+            raise configexc.ValidationError(value, "List cannot contain never!")
+        # Always can't be set with other options
+        elif 'always' in values and len(values) > 1:
+            raise configexc.ValidationError(value,
+                                            "List cannot contain always!")
```

Comment:
I'd prefer such lines to be broken up like this (same for the other `raise` lines):
```python
raise configexc.ValidationError(
    value, "List cannot contain always!")
```